### PR TITLE
enforcement for `<img srcset>`

### PR DESCRIPTION
**Pull request by mikesamuel**
This provides machinery relevant to the proposed solution to
issue #139.

This is not yet wired into the enforcer.

The parser is not security sensitive, but the unparser is.

### CODE REVIEW

File: dist/spec/index.html [hunk 1]
```diff
@@ -1212,9 +1212,9 @@
 		}
 	}
 </style>
-  <meta content="Bikeshed version 9c6ae748322940994300e90a77c34437364b40a6" name="generator">
+  <meta content="Bikeshed version 86d51a2d9c3c0de4540c77189312f7e2393844af" name="generator">
```
Comment (mikesamuel) [inline dist/spec/index.html]: Sorry, this is cruft from running my local install of bikeshed.

**Comment by mikesamuel** [comment]
Handling srcset this way adds a considerable amount of machinery.

**Comment by mikesamuel** [comment]
[Drive by fix](https://github.com/mikesamuel/trusted-types/blob/3808747365aefbce2efee2aececc2737b608abee/src/enforcer.js#L682-L686) for issue #141

**Comment by koto** [comment]
This is deprecated (see #139)

**Pull request closed without merging**